src/target/riscv/riscv.c

```diff
@@ -36,6 +36,7 @@
 #define DTMCONTROL_VERSION			(0xf)
 
 #define DBUS						0x11
+#define REG_NAME_MAX_LENGTH			32u
```

Comment:
Play with this value to see what the smallest we can get away with. Even 8 might be big enough, which would prevent a fair amount of memory waste.

Reply:
Addressed.